### DISCUSSION TITLE
Welcome to specter-desktop Discussions!

### DISCUSSION THREAD

**Discussion by k9ert** [cryptoadvance/specter-desktop#788] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by swimmerkp** [comment]
Hi there, I'm having trouble connecting to my bitcoin core software.  Trying to route over tor, I'm entering the correct rpc user and password.  Pretty sure I'm entering the host info correctly (getting that info from my node).  Maybe it's a port issue since I'm trying to connect over tor?  Using the standard 8332 for mainnet.  Do I need to use a different port?

Oh I forgot to include the error message: process finished with code -1Error message: failed to connect

Update, I found some documentation on my error message and am looking into it.

**Reply by k9ert** [reply]
Have a look into [this document](https://github.com/cryptoadvance/specter-desktop/blob/master/docs/connect-your-node.md) which can help you with this, hopefully.
